### DISCUSSION TITLE
Transaction with date/time and not just date

### DISCUSSION THREAD

**Discussion by msudeshkumar** [firefly-iii/firefly-iii#4314] [Firefly III]
**Description**
- I would like transactions in Firefly III to bear the date and time as well and not just the date.
- Right now I only see that the transaction can only be date stamped. Or am I missing a setting that enables date and time?

**Solution**
Please provide date/time for transactions and not just date.

**Comment by JC5** [comment]
Yes, this is on my list for the new layout.

**Comment by rewri3** [comment]
Please provide date/time for transactions and not just date

**Reply by niklas2810** [reply]
That's exactly the proposed solution?

**Reply by rewri3** [reply]
I'm sorry, the message was sent in error.  However I propose to set the web interface widget in date and time mode.  On the other hand, if you use the waterflyIII application, the app interface is already like this and you can also set the transaction time which is then regularly displayed in the web interface.

**Comment by dmintz7** [comment]
Currently if a transaction is added via API with a time, it is correctly displayed in the application. However when the transaction is updated, the time is removed even if the date is not updated. The update is also recorded in the audit log.

Would this fix this?

**Reply by rewri3** [reply]
Unfortunately not.  I would simply like to use the transaction time in the web interface as well.

**Comment by Soktech** [comment]
I hope that this feature will be available soon because all my current transactions have the incorrect order =(

**Reply by thehijacker** [reply]
Another vote for this. Adding over Abacus for example will add also time, but when I update this transaction over web UI, it will revert to midnight.

**Comment by ilvinc3nt** [comment]
Please, add in web gui 'transaction time' in addition to the date, it'll be amazing.

thanks for this kind of project. <3

**Comment by NotifyOne** [comment]
Hello, you can use the https://github.com/firefly-iii/firefly-iii/pull/8240 or from my repo https://github.com/NotifyOne/firefly-iii/pull/1 for build custom theme and use it
I mount folder /public/v1/js into docker container and it work correctly
![Screenshot from 2023-12-08 23-55-27](https://github.com/firefly-iii/firefly-iii/assets/18601666/018d4b82-7676-40f2-9d62-d30684734d51)

Plan B: use the FIREFLY_III_LAYOUT=v2 in .env (But as for me, it's not suitable for me, I like add more informations)
